perf/micro/immediate-scheduler/observable/from-array.js

```diff
@@ -8,7 +8,7 @@ module.exports = function fromArray(suite) {
     }
 
     var oldFromArrayWithImmediateScheduler = RxOld.Observable.fromArray(args, RxOld.Scheduler.immediate);
-    var newFromArrayWithImmediateScheduler = RxNew.Observable.fromArray(args);
+    var newFromArrayWithImmediateScheduler = RxNew.Observable.from(args);
```

Comment:
Hmm... it looks like perhaps we should leave the other methods in the library, but guide people to use `from` unless they know for a fact their variable is an `Array`.

Again, I'm pretty shocked at the difference a single `Array.isArray` check made here. I've tried a variety of different methods, but I think we're going to have to walk away from this issue/PR